### PR TITLE
Implement Tag and Untag Feature for Video, Module, and Lecture

### LINKED ISSUE #85: As a user, I can add tags to modules
so that I can label and organise my modules more effectively.

### LINKED ISSUE #86: As a user, I can add tags to lectures
so that I can label and organise my lectures more effectively.

### LINKED ISSUE #87: As a user, I can tag videos
so that I can label and organise my videos more effectively.

### PR DESCRIPTION
**Pull request by lennoxtr**
close #85
close #86  
close #87

### CODE REVIEW

**Comment by shaowi** [comment]
please update your code to pass the tests

File: src/main/java/seedu/address/logic/commands/UntagCommand.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    public static final String COMMAND_WORD = "untag";
+
+    //TODO: MODIFY THIS
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Untag a specified video, module, or lecture ";
+    //TODO: MODIFY THIS
+    public static final String MESSAGE_SUCCESS = "Item untagged";
+    public static final String MESSAGE_TAG_NOT_FOUND = "Tag doesn't exist";
+    public static final String MESSAGE_MODULE_NOT_FOUND = "Module doesn't exist in Le Tracker";
+    public static final String MESSAGE_LECTURE_NOT_FOUND = "Lecture doesn't exist in this module";
+    public static final String MESSAGE_VIDEO_NOT_FOUND = "Video doesn't exist in this lecture";
```
Comment (shaowi) [inline src/main/java/seedu/address/logic/commands/UntagCommand.java:37]: Can possibly use the Messages.java instead? If any variable is not available there, you can move them it there

File: src/test/java/seedu/address/logic/parser/ParseArgumentTest.java
Comment (shaowi) [inline src/test/java/seedu/address/logic/parser/ParseArgumentTest.java:1]: Can we remove this file?

File: src/test/java/seedu/address/logic/parser/TagCommandParserTest.java
Comment (shaowi) [inline src/test/java/seedu/address/logic/parser/TagCommandParserTest.java:1]: Same comments as ParseArgumentsTest.java file

**Review by shaowi — CHANGES_REQUESTED** [review]
Please update accordingly to my comments

File: src/test/java/seedu/address/logic/parser/TagCommandParserTest.java
Comment (shaowi) [inline src/test/java/seedu/address/logic/parser/TagCommandParserTest.java:1]: Did u miss out on removing this file?

**Review by shaowi — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/address/logic/parser/TagCommandParserTest.java
Comment (lennoxtr) [reply src/test/java/seedu/address/logic/parser/TagCommandParserTest.java:1]: I removed the file

**Review by shaowi — APPROVED** [review]
LGTM

**Pull request merged**